crates/holochain/src/conductor/conductor.rs

```diff
@@ -394,7 +394,7 @@ mod startup_shutdown_impls {
 
             // We don't care what fx are returned here, since all cells need to
             // be spun up
-            let _ = self.start_paused_apps().await?;
+            let _ = self.start_apps().await?;
```

Comment:
I haven't looked into the code deeply yet but that doesn't seem correct to me here. I don't think we want to start stopped apps when initializing the conductor as these include apps that have been deliberately disabled by the user. Do we have a test that checks that an app can be disabled via the adminwebsocket and won't get automatically re-enabled on conductor restart?

Reply:
I see what you mean. Yeah, this is no longer applicable. Apps can't be paused, so they won't be started when initializing the conductor.